### PR TITLE
feat: icon gallery

### PR DESCRIPTION
**Pull request by theJohnnyMe**
## **Describe pull-request**  
Added icon gallery for easy overview of available icons

## **Issue Linking:**  
- **No issue:** Heard from a user during ScaniaJS meet up as missing feature.

## **How to test**  
1. Check preview link
2. Go to Icons and check "Gallery" story

### CODE REVIEW

File: packages/core/src/stories/foundations/icons-gallery.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            <tds-icon name="${icon}" size="48"></tds-icon>  
+            <p class="tds-detail-05">${icon}</p>  
+        </tds-block>
+ `,
+);
+
+const Template = () =>
+  formatHtmlPreview(
+    `
+    <div style="display: grid; grid-template-columns: repeat(auto-fill, minmax(144px, 1fr)); gap: 16px; padding: 16px;">
```
Comment (mistermalm) [inline packages/core/src/stories/foundations/icons-gallery.stories.tsx:27]: Could be nice to format the HTML string

**Review by mistermalm — APPROVED** [review]

File: packages/core/src/stories/foundations/icons-gallery.stories.tsx (hunk 1 above)
Comment (theJohnnyMe) [reply packages/core/src/stories/foundations/icons-gallery.stories.tsx:27]: I tried some VS code plugins, but it get the same or worse, do you have some tip ?

**Review by nathalielindqvist — APPROVED** [review]
Awesome!

**Review by gokberknur — APPROVED** [review]
Looks really nice! very nice addition :)

**Pull request merged**